stable/elasticsearch/values.yaml

```diff
@@ -29,5 +29,5 @@ DataCpuLimits: "1"
 DataMemoryLimits: "4Gi"
 DataHeapSize: "1536m"
 DataStorage: "30Gi"
-DataStorageClass: "anything"
-DataStorageClassVersion: "alpha"
+# DataStorageClass: "ssd"
+DataTerminationGracePeriodSeconds: 900
```

Comment:
I would prefer to not move the chart to `stable` as long as config values start with an uppercase letter. In this case, I would also switch to nesting first. I think this make sense here, e. g.:

```yaml
client:
  replicas: 2
  port: 2379
  heapSize: "128m"
master:
  replicas: 2
  heapSize: "128m"
data:
  replicas: 3
  storage: "30Gi"
```

See https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md

Generally, I think a move to stable should be a separate PR so it is explicit.

Regarding resources, it's becoming increasingly popular to specify resources like so (without specifying defaults): https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/templates/controller-deployment.yaml#L93